[Graph Vis] Implement UI enhancements

albertoblaz opened the pull request:
## Summary

Closes:
- https://github.com/elastic/kibana/issues/222367

Enhance graph visualization UI with latest Figma update.

### Screenshots

<details><summary>Popover</summary>
<img width="370" alt="Screenshot 2025-06-12 at 10 21 50" src="https://github.com/user-attachments/assets/8060860e-5b93-4d71-b330-1920afb75c6a" />
</details> 

<details><summary>Controls</summary>
<img width="63" alt="Screenshot 2025-06-11 at 17 36 27" src="https://github.com/user-attachments/assets/78b93528-1821-4d77-9536-eb88fd68e3dc" />
</details>

<details><summary>Edges - new color, default to solid stroke, no start marker</summary>
<img width="1078" alt="Screenshot 2025-06-12 at 17 22 25" src="https://github.com/user-attachments/assets/53b46adb-2b79-4c65-ba48-9c74826f2fb0" />
</details>

### Videos

#### Snap nodes to 10px grid

https://github.com/user-attachments/assets/fc732784-1e3b-4277-9bf3-d7a6c9b43f88

#### Zoom / Fit to view transition

https://github.com/user-attachments/assets/8a7627c0-7c00-4321-a05c-ea9fa1910002

### Definition of done

- [x] Update popover container
- [x] Update popover action icons
- [x] Update popover action texts
- [x] Update fit-to-view icons
- [x] Update relationship arrow colors
   - [x] Check if we can increase the current nodes limit - increased to 300
- [x] Snap to Grid: Enable nodes to snap by 10px
- [x] Implement smooth zoom and fit-to-view transitions (200ms duration)
- [x] Remove edge's start marker
- [x] Remove failure representation according to `event.outcome`

### How to test - In Kibana

1. Add this line to your `kibana.dev.yml`:

    ```yml
    uiSettings.overrides.securitySolution:enableGraphVisualization: true
    ```

2. Then, run these 2 commands while running Kibana with `yarn start --no-base-path`. This is for setting up the local env with data.

    ```bash
    node scripts/es_archiver load x-pack/solutions/security/test/cloud_security_posture_functional/es_archives/logs_gcp_audit --es-url http://elastic:changeme@localhost:9200 --kibana-url http://elastic:changeme@localhost:5601
    ```

    ```bash
    node scripts/es_archiver load x-pack/solutions/security/test/cloud_security_posture_functional/es_archives/security_alerts --es-url http://elastic:changeme@localhost:9200 --kibana-url http://elastic:changeme@localhost:5601
    ```

3. Finally in Kibana, go to Alerts and update the date-picker to include data from a year ago. Then check one of the alerts details opening the right-side flyout and find the "Graph preview" section in it.

### How to test - In Storybook

1. Run in terminal:

    ```bash
    yarn storybook cloud_security_posture_graph
    ```

2. Open [http://localhost:9001/](http://localhost:9001/).

### Checklist

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)
- [x] This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.
- [ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed
- [x] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

### Identify risks

No risk, feature is gated under the `securitySolution:enableGraphVisualization` UI setting.

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

albertoblaz committed:
Add 200ms transition to zoom/fit-to-view

albertoblaz committed:
Snap nodes to a 10px grid

albertoblaz committed:
Remove edge's start marker

albertoblaz committed:
Enhance popover

albertoblaz committed:
Enhance actions and controls

albertoblaz committed:
Remove warning notifications

albertoblaz committed:
Remove warning markers

albertoblaz committed:
Default edges stroke to solid style

albertoblaz committed:
Fix border-radius in controls

albertoblaz committed:
Update tests

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/307549)
* Commit: bf57a8a7131f9e2aad955a88ec957f16a3e395cb
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-222830)

### Failed CI Steps
* [FTR Configs #126](https://buildkite.com/elastic/kibana-pull-request/builds/307549#01976025-f2f8-4434-98b6-9a3cf62ec181)
* [FTR Configs #126](https://buildkite.com/elastic/kibana-pull-request/builds/307549#0197604d-e908-4409-988b-9745a363ae92)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/307549#01976025-f2f8-4434-98b6-9a3cf62ec181) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/307549/jobs/01976025-f2f8-4434-98b6-9a3cf62ec181/artifacts/0197604d-ca24-4cd9-a427-49c2d708e221) FTR Configs #<span></span>126 / Cloud Security Posture Security Alerts Page - Graph visualization expanded flyout - filter by node
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/307549#0197604d-e908-4409-988b-9745a363ae92) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/307549/jobs/0197604d-e908-4409-988b-9745a363ae92/artifacts/01976065-7181-4619-ab23-e947407bd8d9) FTR Configs #<span></span>126 / Cloud Security Posture Security Alerts Page - Graph visualization expanded flyout - filter by node

### History
* :broken_heart: [Build #307518](https://buildkite.com/elastic/kibana-pull-request/builds/307518) failed 4734f6f1b937a76d386d268cbde3877248e8486e

cc @albertoblaz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307549","number":307549,"commit":"bf57a8a7131f9e2aad955a88ec957f16a3e395cb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307518","number":307518,"commit":"4734f6f1b937a76d386d268cbde3877248e8486e"}],"number":307549}
buildkite-pr-comment-->

albertoblaz committed:
Update expanded flyout FTR test

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/307644)
* Commit: 19911a0b72046f6e4f08534b98d47ad0d3faddc3
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-222830)

### Failed CI Steps
* [FTR Configs #129](https://buildkite.com/elastic/kibana-pull-request/builds/307644#0197612d-01dd-4fd7-8ab4-69f49bf116e4)
* [FTR Configs #129](https://buildkite.com/elastic/kibana-pull-request/builds/307644#01976150-3ea8-44ca-b5c1-3d04fde59032)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/307644#0197612d-01dd-4fd7-8ab4-69f49bf116e4) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/307644/jobs/0197612d-01dd-4fd7-8ab4-69f49bf116e4/artifacts/01976150-2068-4de5-9447-6f8c7ea80043) FTR Configs #<span></span>129 / Cloud Security Posture Security Alerts Page - Graph visualization expanded flyout - filter by node
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/307644#01976150-3ea8-44ca-b5c1-3d04fde59032) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/307644/jobs/01976150-3ea8-44ca-b5c1-3d04fde59032/artifacts/01976166-bd56-4eba-8e2f-e5bde231fd47) FTR Configs #<span></span>129 / Cloud Security Posture Security Alerts Page - Graph visualization expanded flyout - filter by node

### History
* :broken_heart: [Build #307549](https://buildkite.com/elastic/kibana-pull-request/builds/307549) failed bf57a8a7131f9e2aad955a88ec957f16a3e395cb
* :broken_heart: [Build #307518](https://buildkite.com/elastic/kibana-pull-request/builds/307518) failed 4734f6f1b937a76d386d268cbde3877248e8486e

cc @albertoblaz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307644","number":307644,"commit":"19911a0b72046f6e4f08534b98d47ad0d3faddc3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307549","number":307549,"commit":"bf57a8a7131f9e2aad955a88ec957f16a3e395cb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307518","number":307518,"commit":"4734f6f1b937a76d386d268cbde3877248e8486e"}],"number":307644}
buildkite-pr-comment-->

albertoblaz committed:
Update expanded flyout FTR test

albertoblaz committed:
Update popover icons

albertoblaz committed:
Update edges color

kfirpeled reviewed (commented):
Added some nit comments, overall looks good.

I do have questions,

- I forgot to mention it on the DOD, but I think we update the API to stop distinguish between different event.outcome and to update it not to return warning colors. Makes sense?
   - On the same principle, not to mark entities as danger or warning
- What you think on changing the nodes_limit threshold?

albertoblaz committed:
Turn zoomDuration prop into a constant

albertoblaz committed:
Improve separators in controls

albertoblaz committed:
Remove comment

albertoblaz committed:
Rename icon

albertoblaz committed:
Update translations

albertoblaz committed:
Use constant for grid size

albertoblaz committed:
Restore addWarning

albertoblaz commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/controls/controls.tsx:
```diff
@@ -57,11 +63,14 @@ const CenterLabel = i18n.translate('securitySolutionPackages.csp.graph.controls.
   showFitView = true,
   showCenter = true,
   fitViewOptions,
+  zoomDuration = DEFAULT_ZOOM_DURATION,
```
I thought we'd want to make it customizable, but removed it 👍

albertoblaz commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/controls/controls.tsx:
```diff
@@ -73,72 +82,85 @@ export const Controls = ({
 
   const onZoomInHandler = () => {
-    zoomIn();
+    zoomIn({ duration: zoomDuration });
```
Ignored since we removed `zoomDuration` in favor of an internal constant

albertoblaz commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/controls/controls.tsx:
```diff
@@ -73,72 +82,85 @@ export const Controls = ({
           />
         </EuiFlexItem>
       )}
-      {showZoom && showCenter && <EuiSpacer size="xs" />}
```
Thanks! Great catch! 👍

albertoblaz commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/edge/styles.tsx:
```diff
@@ -87,3 +88,13 @@ export const EdgeLabelOnHover = styled(EdgeLabel)<EdgeLabelProps & EdgeLabelCont
   }
 `;
+
+export const useEdgeColor = (edgeColor: EdgeViewModel['color']) => {
```
I honestly don't see the need, it's a fairly cheap calculation and we only have two colors for now

albertoblaz commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph/graph.tsx:
```diff
@@ -150,7 +150,7 @@ export const Graph = memo<GraphProps>(
           onlyRenderVisibleElements={ONLY_RENDER_VISIBLE_ELEMENTS}
           snapToGrid={true} // Snap to grid is enabled to avoid sub-pixel positioning
-          snapGrid={[1, 1]}
+          snapGrid={[10, 10]} // Snap nodes to a 10px grid
```
Great point, using a constant now!

albertoblaz commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph_investigation/use_entity_node_expand_popover.ts:
```diff
@@ -90,20 +90,20 @@ export const useEntityNodeExpandPopover = (
                   'securitySolutionPackages.csp.graph.graphNodeExpandPopover.showActionsByEntity',
                   {
-                    defaultMessage: 'Show actions by this entity',
+                    defaultMessage: "Show this entity's actions",
```
Didn't know that. I just changed the translation keys so in case you're right, we're covered

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310387)
* Commit: 877c49561e594addaec8c10c70bb67a5022fc23b

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/310387#019787dd-85a5-464f-9071-17cb434122f2)

### History
* :green_heart: [Build #307971](https://buildkite.com/elastic/kibana-pull-request/builds/307971) succeeded 2a9e08c3ffed2562fcea17b8bcf71bfff9f7ad90
* :green_heart: [Build #307705](https://buildkite.com/elastic/kibana-pull-request/builds/307705) succeeded 38bdf8e77955756b574509843b4933c53aa2d188
* :broken_heart: [Build #307644](https://buildkite.com/elastic/kibana-pull-request/builds/307644) failed 19911a0b72046f6e4f08534b98d47ad0d3faddc3
* :broken_heart: [Build #307549](https://buildkite.com/elastic/kibana-pull-request/builds/307549) failed bf57a8a7131f9e2aad955a88ec957f16a3e395cb
* :broken_heart: [Build #307518](https://buildkite.com/elastic/kibana-pull-request/builds/307518) failed 4734f6f1b937a76d386d268cbde3877248e8486e

cc @albertoblaz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310387","number":310387,"commit":"877c49561e594addaec8c10c70bb67a5022fc23b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307971","number":307971,"commit":"2a9e08c3ffed2562fcea17b8bcf71bfff9f7ad90"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307705","number":307705,"commit":"38bdf8e77955756b574509843b4933c53aa2d188"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307644","number":307644,"commit":"19911a0b72046f6e4f08534b98d47ad0d3faddc3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307549","number":307549,"commit":"bf57a8a7131f9e2aad955a88ec957f16a3e395cb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307518","number":307518,"commit":"4734f6f1b937a76d386d268cbde3877248e8486e"}],"number":310387}
buildkite-pr-comment-->

albertoblaz committed:
warning

albertoblaz committed:
update translations

albertoblaz committed:
update copy

albertoblaz committed:
remove translations

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310461)
* Commit: 5deee381bad55820c770da5b99d9027960d5522c
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-222830)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/8c860b79127874df74e0eab6fe73591f8474f55e) | [after](https://github.com/elastic/kibana/commit/5deee381bad55820c770da5b99d9027960d5522c) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 7641 | 7642 | +1 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/8c860b79127874df74e0eab6fe73591f8474f55e) | [after](https://github.com/elastic/kibana/commit/5deee381bad55820c770da5b99d9027960d5522c) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 9.4MB | 9.4MB | +1.5KB |



### History
* :broken_heart: [Build #310387](https://buildkite.com/elastic/kibana-pull-request/builds/310387) failed 877c49561e594addaec8c10c70bb67a5022fc23b
* :green_heart: [Build #307971](https://buildkite.com/elastic/kibana-pull-request/builds/307971) succeeded 2a9e08c3ffed2562fcea17b8bcf71bfff9f7ad90
* :green_heart: [Build #307705](https://buildkite.com/elastic/kibana-pull-request/builds/307705) succeeded 38bdf8e77955756b574509843b4933c53aa2d188
* :broken_heart: [Build #307644](https://buildkite.com/elastic/kibana-pull-request/builds/307644) failed 19911a0b72046f6e4f08534b98d47ad0d3faddc3
* :broken_heart: [Build #307549](https://buildkite.com/elastic/kibana-pull-request/builds/307549) failed bf57a8a7131f9e2aad955a88ec957f16a3e395cb
* :broken_heart: [Build #307518](https://buildkite.com/elastic/kibana-pull-request/builds/307518) failed 4734f6f1b937a76d386d268cbde3877248e8486e

cc @albertoblaz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310461","number":310461,"commit":"5deee381bad55820c770da5b99d9027960d5522c"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310387","number":310387,"commit":"877c49561e594addaec8c10c70bb67a5022fc23b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307971","number":307971,"commit":"2a9e08c3ffed2562fcea17b8bcf71bfff9f7ad90"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307705","number":307705,"commit":"38bdf8e77955756b574509843b4933c53aa2d188"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307644","number":307644,"commit":"19911a0b72046f6e4f08534b98d47ad0d3faddc3"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307549","number":307549,"commit":"bf57a8a7131f9e2aad955a88ec957f16a3e395cb"}],"number":310461}
buildkite-pr-comment-->

kfirpeled committed:
Merge branch 'main' into graph-vis-updates

kfirpeled committed:
actions borealis theme fix - search button badge number layout

kfirpeled committed:
refactor controls fitviewoptions

kfirpeled committed:
- added subdued edge color to align color palette between nodes and edges
- API now don't return warning based on eventOutcome
- Fix layout edge case bug

kfirpeled committed:
refactoring

kfirpeled committed:
added story

kfirpeled commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/controls/actions.tsx:
```diff
@@ -154,6 +154,7 @@ export const Actions = ({
                 css={[
                   css`
                     position: relative;
+                    overflow: visible;
```
`borealis` theme bug fix

kfirpeled commented on x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/controls/controls.tsx:
```diff
@@ -73,72 +82,85 @@ export const Controls = ({
 
   const onFitViewHandler = () => {
-    fitView(fitViewOptions);
+    const options = fitViewOptions
```
Since you are on PTO I took the time to refactor it, take a look and let me know if it makes sense

kfirpeled committed:
updated i18n

kfirpeled committed:
Increased threshold to 300

kfirpeled reviewed (approved):
I applied myself some changes, thank you.@albertoblaz enjoy your time off :))

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/311727)
* Commit: b4651c87166a2c21d43daa7302034dd6105d8fe3
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-222830)

### Failed CI Steps
* [FTR Configs #108](https://buildkite.com/elastic/kibana-pull-request/builds/311727#01979eb3-5d36-4d61-b004-957fee6b3652)
* [FTR Configs #108](https://buildkite.com/elastic/kibana-pull-request/builds/311727#01979ed8-f911-4e3d-bfc5-2187709422f0)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/311727#01979eb3-5d36-4d61-b004-957fee6b3652) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/311727/jobs/01979eb3-5d36-4d61-b004-957fee6b3652/artifacts/01979ed8-e203-4cf3-886c-0fdd693ebb05) FTR Configs #<span></span>108 / Cloud Security Posture Security Alerts Page - Graph visualization expanded flyout - filter by node
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/311727#01979ed8-f911-4e3d-bfc5-2187709422f0) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/311727/jobs/01979ed8-f911-4e3d-bfc5-2187709422f0/artifacts/01979eef-cb71-48f1-a988-8f89c65435f7) FTR Configs #<span></span>108 / Cloud Security Posture Security Alerts Page - Graph visualization expanded flyout - filter by node

### History
* :green_heart: [Build #310461](https://buildkite.com/elastic/kibana-pull-request/builds/310461) succeeded 5deee381bad55820c770da5b99d9027960d5522c
* :broken_heart: [Build #310387](https://buildkite.com/elastic/kibana-pull-request/builds/310387) failed 877c49561e594addaec8c10c70bb67a5022fc23b
* :green_heart: [Build #307971](https://buildkite.com/elastic/kibana-pull-request/builds/307971) succeeded 2a9e08c3ffed2562fcea17b8bcf71bfff9f7ad90
* :green_heart: [Build #307705](https://buildkite.com/elastic/kibana-pull-request/builds/307705) succeeded 38bdf8e77955756b574509843b4933c53aa2d188
* :broken_heart: [Build #307644](https://buildkite.com/elastic/kibana-pull-request/builds/307644) failed 19911a0b72046f6e4f08534b98d47ad0d3faddc3
* :broken_heart: [Build #307549](https://buildkite.com/elastic/kibana-pull-request/builds/307549) failed bf57a8a7131f9e2aad955a88ec957f16a3e395cb

cc @albertoblaz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311727","number":311727,"commit":"b4651c87166a2c21d43daa7302034dd6105d8fe3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310461","number":310461,"commit":"5deee381bad55820c770da5b99d9027960d5522c"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310387","number":310387,"commit":"877c49561e594addaec8c10c70bb67a5022fc23b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307971","number":307971,"commit":"2a9e08c3ffed2562fcea17b8bcf71bfff9f7ad90"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307705","number":307705,"commit":"38bdf8e77955756b574509843b4933c53aa2d188"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/307644","number":307644,"commit":"19911a0b72046f6e4f08534b98d47ad0d3faddc3"}],"number":311727}
buildkite-pr-comment-->

kfirpeled committed:
fixed ftr test

kibanamachine commented:
## Flaky Test Runner Stats
### 🟠 Some tests failed. - [kibana-flaky-test-suite-runner#8428](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/8428)
[❌] x-pack/solutions/security/test/cloud_security_posture_functional/config.ts: 0/25 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F222830%2Fhead)
